app/admin/textrepl/controller.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+import EmberValidations from 'ember-validations';
+
+export default Ember.Controller.extend(EmberValidations, {
+  hideCancelButton: true,
+  updateCapability: 'update_config',
+
+  createExpansion: function() {
+    const newExpansion = this.get('store').createRecord('text-expansion');
+    this.set('newExpansion', newExpansion);
+  }.on('init'),
+
+  validations: {
```

Comment:
generally i add validations at the end @jkleinsc there was discussion of adding a style guide